styleguide/lib/user-menu.md

```diff
@@ -0,0 +1,60 @@
+---
+name: User menu
+category: Components
+---
+
+⚠️ This is a low level building block, and should be automated in a component library, but can be used directly if you want full control.
+
+This is the user menu to be shown when clicking on your image in the header. This component should be linked to the header in a component library.
+
+
+## User Menu with image
+
+```user-menu.html
+<div class="hdd-user-menu">
+  <img class="hdd-user-menu_image" src="http://placekitten.com/150/150"/>
+  <div class="hdd-user-menu_name">
+    Jan Greger Hemb
```

Comment:
Fin informasjon og eksempel, men siden dette er åpent så vurder å bytte til noe mer generisk navn og et mer menneskeaktig bilde.

Reply:
Gjelder forøvrig også headeren

Reply:
Skal se etter bilde